cluster/stone-prd-rh01/tenants/rhtpa-tenant/kustomization.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - release-plan.yaml
+namespace: trusted-content-tenant
```

```suggestion
namespace: rhtpa-tenant
```

This needs to be updated and then it needs a rebuild of the manifests script to pass CI.